cmd/manager/main.go

```diff
@@ -80,6 +85,10 @@ func main() {
 		"feature-gates",
 		"A set of key=value pairs that describe feature gates for alpha/experimental features. "+
 			"Options are:\n"+strings.Join(features.FeatureGates.KnownFeatures(), "\n"))
+	flag.StringVar(&imagePullSecretName, "agent-image-pull-secret", "",
+		"Name of the image pull secret for managed-serviceaccount addon agent serviceaccount, "+
+			"The content of the image pull secret will be copied to the managed cluster."+
+			"This can also be configured with environment variable AGENT_IMAGE_PULL_SECRET.")
```

Comment:
the secret should be in the same namespace of the pod?

Reply:
updated